### PR TITLE
Update renew_certificate to support Nextcloud

### PR DESCRIPTION
**Pull request by dartheide**
If Nextcloud is installed then copy certificates.

### CODE REVIEW

File: renew_certificate.sh [hunk 1]
```diff
@@ -35,3 +35,32 @@ kill -9 $pid || true
...
 
 /etc/init.d/Qthttpd.sh start
+# Nextcloud
+# certificat
+CONF=/etc/config/qpkg.conf
+QPKG_NAME="NextCloud"
+QPKG_ROOT=`/sbin/getcfg $QPKG_NAME Install_Path -f ${CONF}`
+
+### EXPORT
+export QPKG_ROOT
```
Comment (Yannik) [inline renew_certificate.sh]: what is this export necessary for?

File: renew_certificate.sh [hunk 2]
```diff
@@ -35,3 +35,32 @@ kill -9 $pid || true
 rm -rf tmp-webroot
 
 /etc/init.d/Qthttpd.sh start
+# Nextcloud
+# certificat
```
Comment (Yannik) [inline renew_certificate.sh]: please add a new line between the qthttpd.sh start command and your nextcloud stuff, followed by a single line comment "# nextcloud certificate installation".

File: renew_certificate.sh [hunk 3]
```diff
@@ -35,3 +35,32 @@ kill -9 $pid || true
...
+CONF=/etc/config/qpkg.conf
+QPKG_NAME="NextCloud"
+QPKG_ROOT=`/sbin/getcfg $QPKG_NAME Install_Path -f ${CONF}`
+
+### EXPORT
+export QPKG_ROOT
+
+if [ -f $QPKG_ROOT/server.crt ]
+
+then
```
Comment (Yannik) [inline renew_certificate.sh]: please concat lines 47-49 as one line (`if [..]; then`)

**Comment by Yannik** [comment]
Hey @dartheide, could you please review my comments on your pr.

**Comment by dartheide** [comment]
Planning is next week. 

Verstuurd vanaf mijn iPhone

**Comment by Yannik** [comment]
@dartheide Do you still want to get this merged?

**Pull request closed without merging**